opat/govready-0.9.yaml

```diff
@@ -0,0 +1,698 @@
+title: GovReady PBC Accessibility Conformance Report
+product:
+  name: GovReady
+  version: "0.9.1.36"
+  description: GovReady is a user-friendly cybersecurity Governance, Risk, and Compliance (GRC) web application focused on guiding teams through the NIST Risk Management Framework using Compliance-As-Code and collaborative features.
+author:
+  name: Mike Gifford
+  company_name: CivicActions
+  address: 3527 Mt Diablo Blvd, Unit 269, Lafayette, CA 94549
+  email: mike.gifford@civicactions.com
+  phone: (510) 408-7510
+  website: https://civicactions.com/
+vendor:
+  name: Greg Elin
+  company_name: GovReady PBC
+  address: 32011 Crystal Drive STE 400, Arlington, VA 22202-3709
+  email: gregelin@govready.com
+  phone: 202-505-1050
+  website: https://govready.com/
+
+notes: "All user interfaces for the GovReady software are done through the web interface, so the focus is on the WCAG 2.0 Report.  Chapter 5  has been deleted as it is redundant.  GovReady is built on the Django framework and some of the accessibility issues have been inherited from this framework. GovReady is an open source tool with an open issue queue.
+Accessibility issues identified here will be documented as accessibility issues within existing issue queues. Any workarounds will be available within the issue queue until such time as the problem is resolved. The Authoring Tools components, 504, have not been reviewed in this assessment.
+Content is generally saved in a machine readable format that allows for it to be processed in many ways. All GovReady documentation is available through the web interface, so the focus is on the WCAG 2.0 Report.  Chapter 6 has been deleted as it is redundant. The documentation is being served through Read the Docs and with their default theme. Some accessibility issues have been inherited from this framework."
```

Comment:
@mgifford `pre-commit` is complaining about this multi-line string indentation. Do you want to try to indent it or use the multi-line string YAML syntax?

Reply:
Hopefully that caught it.